include/nccl_ofi_param.h

```diff
@@ -181,7 +181,12 @@ OFI_NCCL_PARAM_INT(disable_gdr_required_check, "DISABLE_GDR_REQUIRED_CHECK", 0);
 /*
  * Maximum size of a message in bytes before message is multiplexed
```

Comment:
Also, I know this is existing before this PR, but I think the name `round_robin_threshold` is confusing. A threshold is defined by M-W dictionary as
> a level, point, or value above which something is true or will take place and below which it is not or will not

So the names should really be of the form `<action_when_greater>_threshold`; i.e., before this PR, the variable name should have been `multiplex_threshold`. And after this PR, the two names should be `multiplex_round_robin_threshold` and `multiplex_threshold`. (But you don't need to fix that in this PR.)

Reply:
I think having two variables is the wrong approach here.  We really want a single variable that is size at which we want to start striping, and use msg_size / min_stripe_size to find how many stripes we're going to create, and then work the size out from there.

Reply:
> I think these comments are backward. The first one should say
> 
> ```
> Maximum size of a message in bytes before message is multiplexed in pairs
> ```
> 
> and the second one should say
> 
> ```
> Maximum size of a message in bytes before message is multiplexed over all rails
> ```

I changed the environment variables to be just the min_stripe_size instead of rr_threshold and the mux_rr_threshold

Reply:
> I think having two variables is the wrong approach here. We really want a single variable that is size at which we want to start striping, and use msg_size / min_stripe_size to find how many stripes we're going to create, and then work the size out from there.

Thanks for the feedback. I have modified the code to now calculate the num_stripes to be multiplexed based on this num_of_stripes = min(ceil(msg_size / min_stripe_size), num_of_rails).